priv/static/phoenix_live_reload.js

```diff
@@ -126,8 +126,27 @@ class LiveReloader {
 
   log(level, str){
     let levelColor = level === "debug" ? "darkcyan" : "inherit"
-    let consoleFunc = level === "error" ? level : "log"
-    console[consoleFunc](`%c📡 ${str}`, `color: ${levelColor};`)
+    let consoleFunc = this.logFunc(level)
+    this.logMsg(consoleFunc, str, levelColor)
+  }
+
+  logMsg(fun, str, color) {
+    fun(`%c📡 ${str}`, `color: ${color};`)
+  }
+  
+  logFunc(level){
+    switch(level) {
+      case "debug":
+        return console.debug;
+      case "info":
+        return console.info;
+      case "warning":
+        return console.warn;
+      case "error":
+        return console.error;
+      default:
+        return console.log;
```

Comment:
WDYT about this?

```suggestion
      case "info":
        return console.info;
      case "notice":
        return console.notice;
      case "warning":
        return console.warn;
      case "error":
        return console.error;
      default:
        return console.error;
```

This maps notice to info and everything else is above error anyway.

Reply:
Yeah, I just noticed #150 , I'm fairly new to phx professionally, so... sure!

Reply:
Let me run a quick test with my dev env.

Reply:
Yeah, it works, it is still nice. commit f3c515b